apiserver/facades/agent/uniter/uniter.go

```diff
@@ -1680,60 +1680,44 @@ func (u *UniterAPI) readOneRemoteSettings(ctx context.Context, canAccess common.
 	return settings, nil
 }
 
-func (u *UniterAPI) updateUnitAndApplicationSettingsOp(arg params.RelationUnitSettings, canAccess common.AuthFunc) (state.ModelOperation, error) {
+func (u *UniterAPI) updateUnitAndApplicationSettings(ctx context.Context, arg params.RelationUnitSettings, canAccess common.AuthFunc) error {
```

Comment:
This was transactional when it composed multiple model operations, but it no longer is.